HelpSource/Guides/Debugging-tips.schelp

```diff
@@ -58,6 +58,55 @@ o.free;  // when done, you need to clean up the OSCFunc
 l.flat.plot(numChannels: 2);
 ::
 
+subsection:: Polling
+
+Polling uses a UGen to have the server output values itself.
+
+The simplest way is to use the .poll method, which will output your value 10 times per second.
```

Comment:
Suggested revision: "Polling allows you to debug a SynthDef by printing samples of a UGen's output to the post window. To do this, use the .poll method (a shorthand for the Poll UGen), which prints 10 times per second by default."